reV/supply_curve/supply_curve.py

```diff
@@ -259,6 +259,42 @@ def _parse_trans_table(trans_table):
 
         return trans_table
 
+    @staticmethod
+    def _map_trans_capacity(trans_sc_table):
+        """
+        Map SC gids to transmission features based on capacity. For any SC
+        gids with capacity > the maximum transmission feature capacity, map
+        SC gids to the feature with the largest capacity
+
+        Parameters
+        ----------
+        trans_sc_table : pandas.DataFrame
+            Table mapping supply curve points to transmission features.
+
+        Returns
+        -------
+        trans_sc_table : pandas.DataFrame
+            Updated table mapping supply curve points to transmission features
+            based on maximum capacity
```

Comment:
So this will have connection rows removed where SC capacity > trans capacity, unless SC capacity > all trans max capacity.

Reply:
yes, thats the hope...